installer/bundle/bundle_skel.sh

```diff
@@ -801,6 +802,16 @@ case "$installMode" in
                     [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
                 fi
             done
+			echo "----- Installing MySQL ruby gem -----"
+			set +e
+			$OMSAGENT_RUBY_DIR/gem install mysql2 2>/dev/null
```

Comment:
I don't think these changes belong in the shell bundle at all.

If you want to make installation of the package trigger certain behaviors, you should absolutely use the post-installation scripts for that.

The shell bundle is only to make installation "easy", nothing else. If the customer deploys via their own package management system (common for large customers), they won't use the shell bundle. Instead, they'll extract the packages and put the packages into their package management system.

Please remove these changes from here and instead put them in the postinstallaton scripts for the OMS package. They're different between RPM-based systems and DEB-based systems, although you can still put them in a common location if commands are identical for both.